app/src/main/java/com/mk/tasky/agenda/detail/components/model/ReminderTypes.kt

```diff
@@ -1,9 +1,12 @@
 package com.mk.tasky.agenda.detail.components.model
 
-sealed class ReminderTypes(val type: String) {
-    object TenMinutes : ReminderTypes("10 minutes before")
-    object ThirtyMinutes : ReminderTypes("30 minutes before")
-    object OneHour : ReminderTypes("1 hour before")
-    object SixHours : ReminderTypes("6 hours before")
-    object OneDay : ReminderTypes("1 day before")
+import com.mk.tasky.R
+import com.mk.tasky.core.util.UIText
+
+enum class ReminderTypes(val type: UIText) {
+    TEN_MINUTES(UIText.StringResource(R.string.ten_minutes_before)),
```

Comment:
I think the convention for enums in Kotlin is CamelCase, the uppercase variant is a Java thing I think 🤔 But I'm also not fully sure here

Reply:
The KotlinLang site shows as UPPERCASE, that's why I use it that way
https://kotlinlang.org/docs/enum-classes.html#anonymous-classes